build/Dockerfile

```diff
@@ -3,12 +3,13 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 ENV OPERATOR=/usr/local/bin/noobaa-operator \
     USER_UID=1001 \
     USER_NAME=noobaa-operator
+ARG NOOBAA_BIN_PATH=build/_output/bin/noobaa-operator
```

Comment:
Why ARG and not ENV? 
Are we intending to provide a different path at some point?

Reply:
Actually I added it to make a way for us to build to multi-platform image by mixing `docker manifest` and `docker build`
ARG was chosen over ENV because args can be used to build a conditional multi-stage dockerfile (`if ARG is "aarch64 do ...` kind of) .